src/com/owncloud/android/ui/helpers/FileOperationsHelper.java

```diff
@@ -133,24 +133,62 @@ public void openFile(OCFile file) {
         }
     }
 
+
+
     /**
      * Helper method to share a file via a public link. Starts a request to do it in {@link OperationsService}
      *
      * @param file          The file to share.
      * @param password      Optional password to protect the public share.
      */
-    public void shareFileViaLink(OCFile file, String password) {
+    public void shareFileViaLink(OCFile file,
```

Comment:
We should add comments to the Javadoc of this method for the rest of parameters, as in ```updateShareViaLink```below.